src/okareo/checks.py

```diff
@@ -8,3 +10,43 @@ def evaluate(
         model_output: str, scenario_input: str, scenario_result: str
     ) -> bool | int | float:
         pass
+
+
+class CheckType(Enum):
+    SCORE = "score"
+    PASS_FAIL = "pass_fail"
+
+
+class Check:
+    @abstractmethod
+    def params(self) -> dict:
+        pass
```

Comment:
Can we add docstrings to these? See the following as an illustration of how this is useful in Jupyter
<img width="973" alt="Screenshot 2024-07-01 at 11 28 48 AM" src="https://github.com/okareo-ai/okareo-python-sdk/assets/3316338/391676a0-ba72-4984-85c8-7402d7358f14">